web/src/features/weather-layers/solar/stackBlurImageOpacity.ts

```diff
@@ -1,3 +1,8 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+/* eslint-disable unicorn/no-abusive-eslint-disable */
+/* eslint-disable */
```

Comment:
I thought we where getting rid of these 😅

Reply:
Not for this nasty old code 😉

Reply:
```suggestion
/* eslint-disable */
// @ts-nocheck
```
Shouldn't this be enough? First disable eslint for the whole file which should supress the following ts-check warning?

Reply:
We still need unicorn/no-abusive-eslint-disable it seems, but the ban-ts-comment can be removed by reordering :)